zerver/models.py

```diff
@@ -2840,6 +2840,8 @@ class UserActivityInterval(models.Model):
 
 class UserPresence(models.Model):
     """A record from the last time we heard from a given user on a given client.
+    NOTE: Users can disable updates to this table (see UserProfile.presence_enabled),
+    so this cannot be treated as reliable data on a user's recent activity.
```

Comment:
I think it's worth mentioning what _should_ be used (e.g. UserActivity) in its stead.

Reply:
Makes sense, added an extra sentence